File: BACKEND/backendDietiEstates/src/main/java/backendDietiEstates/backendDietiEstates/Entity/Utente.java
package backendDietiEstates.backendDietiEstates.Entity;

public class Utente {
	private String nome;
	private String cognome;
	private String email;
	private String password;
	
	//Getter and Setter for nome
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	
	//Getter and Setter for cognome
	public String getCognome() {
		return cognome;
	}
	public void setCognome(String cognome) {
		this.cognome = cognome;
	}
	
	//Getter and Setter for email
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	
	//Getter and Setter for password
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}

}

File: FRONTEND/MyApplication/app/src/main/java/com/example/myapplication/Entity/Utente.kt
package com.example.myapplication.Entity

open class Utente {
    private var nome: String? = null
    private var cognome: String? = null
    private var email: String? = null
    private var password: String? = null

    // Getter and Setter for nome
    fun getNome(): String? {
        return nome
    }
    fun setNome(nome: String?) {
        this.nome = nome
    }

    // Getter and Setter for cognome
    fun getCognome(): String? {
        return cognome
    }
    fun setCognome(cognome: String?) {
        this.cognome = cognome
    }

    // Getter and Setter for email
    fun getEmail(): String? {
        return email
    }
    fun setEmail(email: String?) {
        this.email = email
    }

    // Getter and Setter for password
    fun getPassword(): String? {
        return password
    }
    fun setPassword(password: String?) {
        this.password = password
    }
}
